From b776b2bf2b0a41644b6b846ebfb1f72162a3a944 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Wed, 21 Feb 2024 17:56:50 +0200
Subject: [PATCH] [SW-15774] bld: update mgmt package with libpldm

SW-161447 updates the zephyr makefile with regard to the mgmt.
The "split" mgmt package's makefile should be adjusted as well.

Change-Id: Ie973a1b43f7ad6e50cda0251c7a985548205b564
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/package/mgmt/mgmt.mk b/package/mgmt/mgmt.mk
index 8abb87b..240eb3b 100644
--- a/package/mgmt/mgmt.mk
+++ b/package/mgmt/mgmt.mk
@@ -1,6 +1,6 @@
 MGMT_APP_NAME = mgmt
 MGMT_DEPENDENCIES = host-cmake host-lzop host-lz4
-MGMT_APP_MODULES = eeprom mbedtls boot_info i2c-indirect soc_init hl_shell mctp
+MGMT_APP_MODULES = eeprom mbedtls boot_info i2c-indirect soc_init hl_shell mctp pldm
 MGMT_SRC_DIR=$(EMBEDDED_DIR)/zephyr-apps/arcmgmt
 MGMT_ZEPHYR_SRCDIR=$(EMBEDDED_DIR)/zephyr
 
